src/sass_context.cpp

```diff
@@ -375,6 +375,11 @@ extern "C" {
     return options;
   }
 
+  const char* ADDCALL sass_copy_c_string (const char* input_string)
```

Comment:
@mgreter, libsass is actually taking ownership of this string and will free it. This is only for data context's data string only, which could be a large string. In node-sass binding, we are passing the memory to libsass, which libsass free's as part of `sass_delete_data_context`.